### DISCUSSION TITLE
Does standalone binary includes pyfuse3?

### DISCUSSION THREAD

**Discussion by lamyergeier** [borgbackup/borg#8448] [Q&A] [answered]
For example if I use the standalone binary of latest release of Borg 2.0.0.11(borg-linux-glibc236), does that include pyfuse3?

**Comment by ThomasWaldmann** [comment]
I just had a look at the b12 builds:
- bookworm: contains llfuse (visible as `borg.tgz/borg-dir/_internal/llfuse*.so`)
- bullseye: same
- freebsd14: same

**Comment by ThomasWaldmann** [comment]
@lamyergeier why do you ask?

**Reply by lamyergeier** [reply]
I was not sure if it contains llfuse or pyfuse3 as llfuse is not recommended.

**Reply by ThomasWaldmann** [reply, marked as answer]
It both works.
